src/main/java/com/missafternoon/budjen/transactions/JSONInterpreter.java

```diff
@@ -0,0 +1,44 @@
+package com.missafternoon.budjen.transactions;
+
+import com.fasterxml.jackson.core.JsonGenerationException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.IOException;
+import java.util.List;
+
+public class JSONInterpreter {
+
+    private ObjectMapper mapper;
+
+    public JSONInterpreter() {
+        mapper = new ObjectMapper();
+    }
+
+    public String convertToString(List<Transaction> transactions) {
+        try {
+            return mapper.writeValueAsString(transactions);
+        } catch (JsonGenerationException e) {
+            e.printStackTrace();
```

Comment:
It might make things clearer if you just catch an `IOException` and wrap the exception in a `RuntimeException` like so:

```suggestion
            throw new RuntimeException(e);
```

Your method then become a one-liner try and a one-liner catch, and you still see all the information in the stacktrace. In fact, `e.printStackTrace();` shouldn't be used ever really. It's just a silly autocomplete that the IDE does for you.